### ISSUE TITLE
Report Comparer Diff Alerting

### ISSUE THREAD

**Issue by JeridInsight** [tabulartools/pbi-explorer#26]
### The feature request

In the Report Comparer, it delineates a "diff" by coloring the property red or green.  But these are only visible when you are fully drilled down on that object's properties.  

For instance, there are diffs associated with this visualization, but you can't tell unless you drill into it:

![image](https://github.com/tabulartools/pbi-explorer/assets/105085417/0cff56ce-4c7a-4f81-8a23-799a6f81fd19)

![image](https://github.com/tabulartools/pbi-explorer/assets/105085417/dc0649da-fb92-4f2f-a6f2-3ea674edec03)

This might not be a big deal with smaller reports, as a Full Expand would reveal all the diffs and you'd be able to see them, but with a bigger report with lots of objects / properties, you would really have to drill through to find them.  

I know that turning off the Show Identical Items will essentially filter out all the noise and only show you the diffs, but it would be nice to have a visual representation that alerts you to the fact that an object in the hierarchy has a diff "inside" of it. 

### Proposed solution

Add between the two Object / Property labels (in the area where the Refine Comparison drop-down lives) a Total Diffs count.  And then on above each report's Object / Property pane a dedicated Total Adds, Total Deletes and Total Mods count.   Then in the Object / Property tree, change the color of the text of the object that contains an Add, Delete, or Mod.  Maybe different colors for the combinations, as an object could contain an Add and a Delete, or an Add and a Mod, etc....  Or just bold the ones that contain a diff to make it simple?

**Comment by DBojsen** [comment]
Hi @JeridInsight 

We already have visual indicators that show that there is changes to a child object

![271016565-0cff56ce-4c7a-4f81-8a23-799a6f81fd19](https://github.com/tabulartools/pbi-explorer/assets/22237520/d24ce142-b5e2-43a6-9297-9f2620f8542d)

**Comment by JeridInsight** [comment]
Yep!  You're right.  I don't know how I missed those.  Hahaha...

What do you think about the Total Diffs, Total Adds, Total Deletes, Total Mods?

**Comment by DBojsen** [comment]
I like the idea, and code wise it will be easy to implement - it's just a matter of not cluttering the UI to much.
We'll keep your suggestion in mind.
